internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -148,6 +144,15 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 		os.Exit(1)
 	}
 
+	if err := mgr.AddHealthzCheck("health", healthz.Ping); err != nil {
```

Comment:
```suggestion
	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
```

Reply:
We should not add these endpoints. I will push a fix to upstream/kubebuilder. 
See that the scaffolds are working because we are using the default ones which are provided by controller-runtime which is the goal here; https://github.com/kubernetes-sigs/controller-runtime/blob/3410e75b087895ff6cd79b6c28dd31c2145c813d/pkg/manager/internal.go#L299

Reply:
Ah not sure how I missed [this](https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/main.go#L139-L146) in review.